infrastructure/messaging/application/middleware/logging_middleware.py

```
@@ -0,0 +1,59 @@
+                "Request completed",
+                extra={
+                    "request_id": request_id,
+                    "method": request.method,
+                    "path": request.url.path,
+                    "status_code": response.status_code,
+                    "process_time_ms": round(process_time * 1000, 2)
+                }
+            )
+
+            response.headers["X-Request-ID"] = request_id
+            return response
```

Ruff (0.15.11)

[warning] Consider moving this statement to an `else` block

(TRY300)